src/test/java/com/hazelcast/kubernetes/ServiceEndpointResolverTest.java

```diff
@@ -73,22 +73,31 @@ public void setup() throws Exception {
 
     @Test
     public void resolveWithNamespaceAndNoNodeInNamespace() {
-        ServiceEndpointResolver sut = new ServiceEndpointResolver(LOGGER, SERVICE_NAME, null, null, NAMESPACE, null, KUBERNETES_MASTER_URL, API_TOKEN);
+        ServiceEndpointResolver sut = new ServiceEndpointResolver(LOGGER, SERVICE_NAME, 0, null, null, NAMESPACE, null, KUBERNETES_MASTER_URL, API_TOKEN);
         List<DiscoveryNode> nodes = sut.resolve();
 
         assertEquals(0, nodes.size());
     }
 
     @Test
     public void resolveWithNamespaceAndNodeInNamespace() {
+        resolveWithNamespaceAndNodeInNamespace(0, 1);
```

Comment:
Why 1, shouldn't it be 5701 ?

Reply:
port value 1 comes from kubernetes discovery result in this test, from property 'hazelcast-service-port' which is set in method createEndpoints()

Reply:
Good to know, it was not immediately apparent! Maybe worth adding a comment?